tests/test_type_adaptor.py

```diff
@@ -0,0 +1,118 @@
+"""Tests for the GraphQL Response Parser.
+
+At the moment we use the Star Wars schema which is fetched each time from the
+server endpoint. In future it would be better to store this schema in a file
+locally.
+"""
+import copy
+from gql.type_adaptor import TypeAdaptor
+import pytest
+import requests
+from gql import Client
+from gql.transport.requests import RequestsHTTPTransport
+
+class Capitalize():
+    @classmethod
+    def parse_value(self, value: str):
+        return value.upper();
+
+@pytest.fixture
```

Comment:
Yes, local schema would be best.

The project already depends on this URL (the same pattern is used elsewhere to get the schema), and we found that because of CSRF tokens it was quite cumbersome to get the schema via the standard tools (I tried the graphql-cli, and a json-to-graphql converter). So I left it like this to avoid wasting more time.